frontend/src/components/widgets/TimeInput/TimeInput.tsx

```diff
@@ -155,7 +155,7 @@ class TimeInput extends PureComponent<Props, State> {
 
     return (
       <div className="stTimeInput" style={style}>
-        <StyledWidgetLabel>
+        <WidgetLabel visible={!!element.label}>
```

Comment:
Do we want to consider/support the case where a widget has an empty label but still has a tooltip?

I would think it should be pretty rare, but it does seem like it may be something that people might occasionally want that isn't possible for all widgets using the new `WidgetLabel` wrapper since all of them have a `visible` attribute based only on the truthiness of `element.label`.

Reply:
I think (?) this isn't really a problem in reality since when the widget is coming from Python then element.label is never falsy. At most it can be the empty string, as its type in the proto is `string`.

The only time this could be an issue is when we use a widget directly from JS, such as in the Settings dialog. But in that case we have full control over the code and can refactor it if the need ever arises.